packages/react/src/themes/teams/components/Embed/embedVariables.ts

```diff
@@ -0,0 +1,13 @@
+export interface EmbedVariables {
+  controlColor: string
+  width: number
```

Comment:
Not sure if this is the right typing, as I would expect the user to provide something like: `10px` or `1rem` which is not a number.

Reply:
Great catch, updated 👍